builder/Dockerfile.ubuntu-2204

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:jammy
+
+ENV OS_IDENTIFIER ubuntu-2204
+
+RUN set -x \
+  && sed -i "s|# deb-src|deb-src|g" /etc/apt/sources.list \
+  && export DEBIAN_FRONTEND=noninteractive \
+  && apt-get update \
+  && apt-get install -y libblas-dev libcurl4-openssl-dev libicu-dev liblapack-dev libpcre2-dev wget python3-pip ruby ruby-dev \
+  && apt-get build-dep -y r-base
+
+RUN pip3 install awscli
+
+# Pin fpm to avoid git dependency in 1.12.0
+RUN gem install fpm:1.11.0
```

Comment:
I ran into an issue with pinning fpm to 1.11.0 for Debian 11 (https://github.com/rstudio/r-builds/pull/106), and see the same error on Ubuntu 22:
```sh
...
Traceback (most recent call last):
	9: from /usr/local/bin/fpm:25:in `<main>'
	8: from /usr/local/bin/fpm:25:in `load'
	7: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/bin/fpm:7:in `<top (required)>'
	6: from /var/lib/gems/2.7.0/gems/clamp-1.0.1/lib/clamp/command.rb:133:in `run'
	5: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/command.rb:574:in `run'
	4: from /var/lib/gems/2.7.0/gems/clamp-1.0.1/lib/clamp/command.rb:68:in `run'
	3: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/command.rb:487:in `execute'
	2: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/package/deb.rb:494:in `output'
	1: from /var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/package/deb.rb:494:in `tap'
/var/lib/gems/2.7.0/gems/fpm-1.11.0/lib/fpm/package/deb.rb:495:in `block in output': uninitialized constant FPM::Package::Deb::Zlib (NameError)
```
I ended up installing the latest (1.13.1 at the time) it to get it working, but it's kind of a headache either way. Both pinned and latest installs have caused issues in the past.